src/types/intellect.ts

```diff
@@ -0,0 +1,58 @@
+import { Division, Subdivision } from './ecampus';
+import { AcademicDegree, AcademicStatus } from './enums';
+
+export type Teacher = {
+    profile: string;
+    credo: string;
+    photo: string;
+    fullName: string;
+    userIdentifier: string;
+    id: number;
+    isConfirmed: boolean;
```

Comment:
I think isConfirmed has no sense on client. This used only for filtering on backend